Views/UIKit/StatusBodyView.swift

```diff
@@ -149,6 +176,9 @@ extension StatusBodyView {
             if configuration.showContentToggled && !identityContext.identity.preferences.readingExpandSpoilers {
                 height += .compactSpacing
                 height += contentHeight
+            } else if viewModel.identityContext.appPreferences.foldLongPosts && contentHeight / contentFont.lineHeight > Self.numLinesBeforeFolding {
+                height += .compactSpacing
+                height += contentFont.lineHeight
```

Comment:
Not 100% sure how we might get the exact height of a one-line excerpt… get the first line and then `.height(width: .infinity, font: contentFont)` maybe?

Reply:
Sorry to leave this comment hanging. I saw it, but didn't know the answer.